Makefile

```diff
@@ -64,8 +65,9 @@ dynbinary: ## build dynamically linked binary
 
 vendor: vendor.conf ## check that vendor matches vendor.conf
 	rm -rf vendor
-	bash -c 'vndr |& grep -v -i clone'
+	bash -c 'vndr |& grep -v -i clone | tee /tmp/vndr.log'
```

Comment:
Perhaps we should store this in `./build/` as well 🤔

Reply:
sure, makes sense. Will update it.

Reply:
I see build dir doesn't exist in ci server. It fails with 'no such directory'.
I added it to the project's root dir and also added the file in .gitignore, .dockerignore